pkg/pulumiyaml/run_test.go

```diff
@@ -1868,59 +1864,6 @@ resources:
 	})
 }
 
-func TestGetPulumiConfNodes(t *testing.T) {
```

Comment:
Should we not be testing  getConfNodesFromMap as a replacement?

Reply:
Yeah probably.  Done :heavy_check_mark: